src/Bundler/EntryPoint.php

```diff
@@ -15,27 +15,37 @@
 final class EntryPoint
 {
     private $file;
-    private $bundle_files;
-    private $vendor_files;
-    private $asset_files;
+    private $files       = [];
+    private $asset_files = [];
 
-    public function __construct(DependencyNodeInterface $file)
+    public function __construct(DependencyNodeInterface $file, callable $split_strategy = null)
```

Comment:
Using generics it would be possible to denote the argument and return type of this callable
```c#
Func<string, string?> splitStrategy
```

PHP can't do this, but we can do something similar if we'd create an interface with one method.